.vsts-ci/install-ps.yml

```diff
@@ -48,19 +48,31 @@ phases:
     jobName: InstallPowerShellUbuntu
     pool: ubuntu-latest
     verification: |
-      if ([Version]"$($PSVersionTable.PSVersion.Major).$($PSVersionTable.PSVersion.Minor).$($PSVersionTable.PSVersion.Patch)" -lt [version]"6.2.0")
+      if ([Version]"$($PSVersionTable.PSVersion.Major).$($PSVersionTable.PSVersion.Minor).$($PSVersionTable.PSVersion.Patch)" -lt [version]"7.3.0")
       {
         throw "powershell was not upgraded: $($PSVersionTable.PSVersion)"
       }
 
+- template: templates/install-ps-phase.yml
+  parameters:
+    scriptName: sudo ./tools/install-powershell.sh
```

Comment:
I'm confused by this - do we actually get the version from the _current_ installed pwsh or the new one? and will this script be executable in the container?

Reply:
verification runs after we run the install script.  It's possible it's the old pwsh, but the verification is only intended to catch issues that we have had in the past, which is mainly that the installation left the system in a bad state.